superset/assets/javascripts/explore/exploreUtils.js

```diff
@@ -65,5 +93,33 @@ export function getExploreUrl(form_data, endpointType = 'base', force = false,
     });
   }
   uri = uri.search(search).directory(directory);
-  return uri.toString();
+  const payload = { ...formData };
+
+  return {
+    url: uri.toString(),
+    payload,
+  };
+}
+
+export function exportChart(formData, endpointType) {
```

Comment:
seems like a mix of positional + named in this PR, big fan of consistently using the latter (which is used in `getExploreUrlAndPayload` for instance) but not a blocker.